cli/dcoscli/tables.py

```diff
@@ -441,20 +462,328 @@ def id_and_containers(pod):
+        """Extracts the value parameter from given row entry. If value
+        is not present, EMPTY_LINE will be returned
+
+        :param entry: row entry
+        :type entry: [dict]
+        :param value: value which should be extracted
+        :type value: string
+        :rtype: str
+        """
+        return entry.get('processedOffersSummary', {}).get(value, EMPTY_ENTRY)
+
+    key_column = 'ID'
+    fields = OrderedDict([
+        (key_column, lambda entry:
+            entry.get('app', entry.get('pod', {})).get('id', EMPTY_ENTRY)
+         ),
+        ('SINCE', lambda entry:
+            entry.get('since', EMPTY_ENTRY)
+         ),
+        ('INSTANCES TO LAUNCH', lambda entry:
+            entry.get('count', EMPTY_ENTRY)
+         ),
+        ('OVERDUE', lambda entry:
+            entry.get('delay', {}).get('overdue', EMPTY_ENTRY)
+         ),
+        ('PROCESSED OFFERS', lambda entry:
+            extract_value_from_entry(entry, 'processedOffersCount')
+         ),
+        ('UNUSED OFFERS', lambda entry:
+            extract_value_from_entry(entry, 'unusedOffersCount')
+         ),
+        ('LAST UNUSED OFFER', lambda entry:
+            extract_value_from_entry(entry, 'lastUnusedOfferAt')
+         ),
+        ('LAST USED OFFER', lambda entry:
+            extract_value_from_entry(entry, 'lastUsedOfferAt')
+         ),
+    ])
+
+    tb = table(fields, queued_apps, sortby=key_column)
+    tb.align[key_column] = 'l'
+    tb.align['SINCE'] = 'l'
+    tb.align['INSTANCES TO LAUNCH'] = 'l'
+    tb.align['OVERDUE'] = 'l'
+    tb.align['PROCESSED OFFERS'] = 'l'
+    tb.align['UNUSED OFFERS'] = 'l'
+    tb.align['LAST UNUSED OFFER'] = 'l'
+    tb.align['LAST USED OFFER'] = 'l'
+
+    return tb
+
+
+def queued_app_table(queued_app):
+    """Returns a PrettyTable representation of the Marathon
+    launch queue content.
+
+    :param queued_app: app to render
+    :type queued_app: dict
+    :rtype: PrettyTable
+    """
+
+    def calc_division(dividend, divisor):
+        """Calcs divident / divisor, displays 0 if divisor equals 0.
+
+        :param dividend: divident
+        :type dividend: int
+        :param divisor: divisor
+        :type divisor: int
+        :rtype: str
+        """
+        if divisor == 0:
+            return 0
+        else:
+            return 100 * dividend / divisor
+
+    fields = OrderedDict([
+        ('RESOURCE', lambda entry:
+            calculations.get(entry, {}).get('RESOURCE', EMPTY_ENTRY)
+         ),
+        ('REQUESTED', lambda entry:
+            calculations.get(entry, {}).get('REQUESTED', EMPTY_ENTRY)
+         ),
+        ('DECLINED', lambda entry:
+            calculations.get(entry, {}).get('DECLINED', EMPTY_ENTRY)
+         ),
+        ('DECLINED PERCENTAGE', lambda entry:
+            calculations.get(entry, {}).get('DECLINED PERCENTAGE', EMPTY_ENTRY)
+         ),
+    ])
+
+    """Make sure only display this table if api offers according information"""
```

Comment:
if we don't have this information, we should display N/A in each column, or output "This command is not supported on your cluster"